r/tools/nixlibs.R

```diff
@@ -371,9 +371,17 @@ ensure_cmake <- function() {
   if (is.null(cmake)) {
     # If not found, download it
     cat("**** cmake\n")
-    CMAKE_VERSION <- Sys.getenv("CMAKE_VERSION", "3.19.2")
+    CMAKE_VERSION <- Sys.getenv("CMAKE_VERSION", "3.21.4")
     if (tolower(Sys.info()[["sysname"]]) %in% "darwin") {
       postfix <- "-macos-universal.tar.gz"
+    } else if (tolower(Sys.info()[["machine"]]) == "arm64") {
+      postfix <- "-linux-aarch64.tar.gz"
+    } else if (tolower(Sys.info()[["machine"]]) == "armv7l") { # e.g. Raspberry Pi OS
+      # There are no binary available for this architecture
+      stop(paste0(
+         "*** cmake was not found locally.",
+         "    Please make sure cmake >= 3.10 is installed and available on your PATH."
+      ))
     } else {
       postfix <- "-Linux-x86_64.tar.gz"
```

Comment:
It seems `-linux-x86_64.tar.gz` is right. https://github.com/Kitware/CMake/releases/tag/v3.21.4

Reply:
👍 thanks, looks like that has changed since the last time we bumped the version: https://github.com/Kitware/CMake/releases/tag/v3.19.2

Reply:
Sorry, which bit of logic are you referring to?

Reply:
Aha, the uppercase/lowercase 'l'?  Thanks!